src/components/video-player/useVideoContainerStyles.ts

```diff
@@ -23,9 +23,9 @@ export const useVideoContainerStyles = makeStyles()(theme => ({
  * A style that simulate video's FileCard sizes used in stories
  * https://github.com/Collaborne/carrot-app-ui/blob/66f02a36cb9182b139a5f701867e8bd8b6ddb9ca/src/components/cards/FileCard.tsx#L148
  * */
-export const useFilePlayerStyles = makeStyles()(theme => ({
+export const useFilePlayerStyles = makeStyles()({
```

Comment:
Please remove the comment, which doesn't fit any longer (_"simulate video's FileCard sizes"_)